simulator/src/css/main.stylesheet.css

```diff
@@ -1777,8 +1777,12 @@ input[type='range']::-webkit-slider-thumb {
 
 
 .code-window .CodeMirror {
-  height: calc(100% - 78px);
-  overflow: scroll;
+  height: calc(100vh - 72px);
```

Comment:
@tachyons @ayan-biswas0412 
Should I set the height dynamically in Javascript, so the position of the horizontal scrollbar won't be affected in the future with a change in height of `navbar` or `tabsbar`

Reply:
cc @nitin10s